k6/config.ts

```diff
@@ -0,0 +1,54 @@
+export const BASE_URL = "http://localhost:8998";
```

Comment:
Can we take this url from env?

Reply:
Yeah, we should be able to.

Reply:
This will take some setup, we will look into this in V2.

Reply:
Okay.👍